### DISCUSSION TITLE
Framework

### DISCUSSION THREAD

**Discussion by Tunklj** [recodehive/Stackoverflow-Analysis#464] [Q&A] [answered]
express or nestjs which is suitable for big project and can maitain better in the future?

**Comment by quoc10973** [comment, marked as answer]
Express.js:
Lightweight, flexible, and minimalistic.
Allows custom project architecture, but lacks built-in structure, which can lead to inconsistencies in large projects.
Scalability is possible, but requires manual setup and third-party libraries for advanced features.
NestJS:
Built on top of Express, providing a modular and opinionated architecture.
Enforces best practices (like SOLID principles, dependency injection), making it easier to maintain and scale.
Ideal for large, complex projects as it provides structure and built-in features.
So big project should use Nestjs
